.github/CODEOWNERS

```diff
@@ -27,7 +27,7 @@
 /src/workflows-app/ @DataBiosphere/broad-workflow-management @DataBiosphere/broad-workflow-execution
 /src/workspace-data/ @DataBiosphere/analysisjourneys
 /integration-tests/tests/billing-projects.js @DataBiosphere/broadworkspaces-terra-ui
-/integration-tests/tests/janitor.js @DataBiosphere/broadworkspaces-terra-ui
+/integration-tests/tests/day-janitor.js @DataBiosphere/broadworkspaces-terra-ui
```

Comment:
I find the name "day-janitor" confusing because this doesn't just run in the daytime… it runs whenever the integration tests run. 

Currently the only thing that this file does is attempt to delete orphaned test workspaces. Why don't we rename it to `delete-orphaned-workspaces`, and if it ever grows to more than that we can generalize the name?

Reply:
Yeah, that's a great idea! I will change the name.